cirq/value/random_state.py

```diff
@@ -12,20 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import cast, Optional, Union
+from typing import cast, Any, Optional, Union
 
 import numpy as np
 
-RANDOM_STATE_LIKE = Optional[Union[np.random.RandomState, int]]
+RANDOM_STATE_LIKE = Optional[Union[np.random.RandomState, int, Any]]
 
 
 def parse_random_state(random_state: RANDOM_STATE_LIKE
                       ) -> np.random.RandomState:
+    """Interpret an object as the state of a pseudorandom number generator.
```

Comment:
Add `Args` and `Returns` to this docstring

Reply:
Done.